pkg/locspec/locations.go

```diff
@@ -488,10 +488,20 @@ func SubstitutePath(path string, rules [][2]string) string {
 	if strings.Contains(path, "\\") { //dependent on the path
 		separator = "\\"
 	}
+
+	// First try with exact match
```

Comment:
I think this check should be part of the normal loop, because we don't do best match.

Reply:
Sounds good, I've updated the PR to reflect this behavior